.github/workflows/github_pages.yml

```diff
@@ -37,25 +37,63 @@ on:
         default: ""
         description: "The domain to be prepended to feed URLs (Pelican's FEED_DOMAIN setting). If not passed this will default to the URL of your GitHub Pages site, which is correct in most cases."
         type: string
+      deploy:
+        required: false
+        default: true
```

Comment:
`deploy` defaults to `true` for backwards-compatibility with existing caller workflows that're already calling `github_pages.yml` and not passing any `deploy` input: we want those workflows to continue deploying the site not just building it or crashing.

Reply:
I like this approach a lot better. Let me test for my site in a PR and see if it works properly :)